tests/Aspire.Hosting.Kafka.Tests/KafkaFunctionalTests.cs

```diff
@@ -139,6 +160,9 @@ await pipeline.ExecuteAsync(async token =>
                 {
                     // Stops the container, or the Volume/mount would still be in use
                     await app.StopAsync();
+                    //kafka shutdown has delay,so without delay to running instance using same data and second instance failed to start.
+                    await Task.Delay(TimeSpan.FromMinutes(1));
```

Comment:
Is there a way we can signal when this happens? Adding a time based delay is a recipe for flakey tests.

Reply:
We have https://github.com/dotnet/aspire/issues/4878 for this but for the temporary solution, we could use `docker inspect` to check the container state but I couldn't find the container name in the app-model.

Reply:
I think we might be able to improve this situation in these tests scenarios. I’ll take a look.

Reply:
Filed #5714